tests/jvm-log4j-backend-our-context/build.gradle.kts

```diff
@@ -31,6 +31,12 @@ plugins {
 dependencies {
     testImplementation(project(":logging"))
     testImplementation(project(":fixtures"))
+
+    /**
+     * Backend and context should have been added to [runtimeOnly].
+     * But tests make sure that the platform indeed picked up the expected
```

Comment:
Again, to my understanding, a few steps are skipped in this explanation. Please expand.